pos_event_sale/static/src/js/popups/EventTicketsPopup.js

```diff
@@ -54,6 +54,19 @@ odoo.define("pos_event_sale.EventTicketsPopup", function(require) {
             );
         },
 
+        getTicketFinalPrice: function(ticket) {
+            const pricelist = _.findWhere(self.posmodel.pricelists, {
+                name: this.pos.get_order().pricelist.name,
+            });
```

Comment:
```suggestion
            const pricelist = this.pos.get_order().pricelist;
```
This should work, no? The pricelist reference is right there